LiteCore/Support/SecureRandomize.hh

```diff
@@ -38,6 +38,51 @@
     }
 
     #define SECURE_RANDOMIZE_AVAILABLE 1
+    
+#elif defined(_CRYPTO_MBEDTLS)
+
+    #include <mbedtls/entropy.h>
+    #include <mbedtls/ctr_drbg.h>
+
+#ifdef MBEDTLS_NO_PLATFORM_ENTROPY
+    #include <Windows.h>
```

Comment:
This could probably be better, but for our supported platform list it is OK.  It is manually defined by CMake.  I'll make a more robust way that will account for other users though.